### ISSUE TITLE
Error page middleware returns an empty body on mono

### ISSUE THREAD

**Issue by Praburaj** [aspnet/Diagnostics#27]
ErrorPage middleware where ShowExceptionDetails flag is turned on returns an empty page in mono. For example the below sample application. Turning this flag on get the page working, but exception information is the critical part.  

```
public void Configure(IBuilder app)
        {
			app.UseErrorPage(new ErrorPageOptions()
				{
					ShowCookies = true,
					ShowEnvironment = true,
					ShowExceptionDetails = false, //Turn this on to see an empty body.
					ShowHeaders = true,
					ShowQuery = true,
					ShowSourceCode = true,
					SourceCodeLineCount = 10
				});

			app.Run (async context => {
				throw new System.Exception("Hello world");
				await context.Response.WriteAsync("Hello world");
			});
}
```
